packages/loot-core/src/mocks/random.ts

```diff
@@ -0,0 +1,18 @@
+import * as Platform from '../client/platform';
+
+// Fake "random" function used to have stable data structures for
+// e2e and visual regression tests
+let pseudoRandomIterator = 0;
+function pseudoRandom(): number {
+  pseudoRandomIterator += 0.45;
+
+  if (pseudoRandomIterator > 1) {
+    pseudoRandomIterator = 0;
+  }
+
+  return pseudoRandomIterator;
+}
```

Comment:
I understand the problem with random values in VRT, but it's unclear to me how this gives stability. 
Do the tests have to run in the same order every time for this to produce consistent values? (maybe that's a guarantee in our e2e setup? no parallelization or randomization of test order?)

Reply:
So this `random.ts` file is used for the test data that is generated when clicking on "create test data" button in the app.

Before: we used `Math.random` for transaction counts, categories, amounts, etc.

After: we use this fake randomization function when running e2e and vrt tests. Thus the output is always stable for all the test runs. The tests don't need to run in the same order - they can be ran in any order we want.

If you want to actually mimic this in the demo app: you can set the useragent to `playwright`. Then you'll get the same environment as e2e tests would.

Reply:
Ahh, that makes sense. 👍🏻